src/tools/publications.ts

```diff
@@ -190,8 +198,14 @@ ${r.content}`;
         .describe(
           "Full content of the publication. Use [{ref}] or [{ref},{ref}] inlined in content for citations.",
         ),
+      attachment_path: z
+        .string()
+        .optional()
+        .describe(
+          "Optional path to a file or directory in your computer to attach to the publication. Requires the computer tool to be active, otherwise this parameter is ignored.",
```

Comment:
Let's do an array of paths to files only. No directory support.

> Requires the computer tools to be active, otherwise this parameter is ignored.

I think you can probably figure that out and inject the tool only if the computer tools are available.